include/ismrmrd/serialize.h

```diff
@@ -240,37 +241,62 @@ void archive_data(Archive &ar, uint16_t data_type, void *data, size_t datasize)
 }
 
 template <class Archive>
-void save(Archive &ar, ISMRMRD::ISMRMRD_Image const &image, const unsigned int version, unsigned int precision, float tolerance) {
+void save(Archive &ar, ISMRMRD::ISMRMRD_ImageHeader const &hdr, void* data, size_t data_sz, char* attribute_string,
+          size_t attr_str_sz, const unsigned int version, bool active, unsigned int precision, float tolerance) {
     if (ISMRMRD_SERIALIZE_VERSION != version)
```

Comment:
You need to bump the serialize version, because of the change in the compression parameters.